lumigator/python/mzai/backend/backend/api/routes/health.py

```diff
@@ -46,9 +51,24 @@ def get_job_metadata(job_id: UUID) -> JobSubmissionResponse:
             detail="Invalid JSON response"
         ) from e
 
+@router.get("/jobs/{job_id}/logs")
+def get_job_logs(job_id: UUID) -> JobLogsResponse:
+    resp = requests.get(ray_jobs() + f"{job_id}/logs")
+    if resp.status_code == 200:
```

Comment:
```suggestion
    if resp.status_code == HTTPStatus.OK:
```

Reply:
I'll change the general approach to errors to fit your current impl in getting jobs above.